src/components/pages/StudentHome/index.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react';
+import { Layout } from 'antd';
+import 'antd/dist/antd.css';
+import Banner from '../../common/Banner';
+import StudentSideBar from './StudentSidebar';
+import StudentDashboard from './StudentDashboard';
+import '../../../styles/ParentStyles/index.less';
+
+function StudentHome() {
+  const { Content, Sider } = Layout;
+  const [collapsed, setCollapsed] = useState(false);
+
+  const toggleCollapsed = () => {
+    setCollapsed(!collapsed);
+  };
+
+  return (
+    <div className="App">
+      <Layout style={{ width: '100%' }}>
+        <Sider
+          collapsible
+          collapsed={collapsed}
+          onCollapse={toggleCollapsed}
+          theme="light"
+        >
+          <div style={{ display: 'flex', justifyContent: 'center' }}></div>
+          <StudentSideBar />
+        </Sider>
```

Comment:
Could be wrong but I believe when you use Sider in your StudentSidebar already you don't need to also add it here.

Reply:
Seconding Brandon here as well - I've tested myself and repeating your sider here isn't necessary. 😄

Reply:
I cleaned that up just now.